code/game/objects/items/control_wand.dm

```diff
@@ -33,13 +60,84 @@
 		if(WAND_EMERGENCY)
 			mode = WAND_OPEN
 	update_icon_state()
-	balloon_alert(user, "mode: [desc[mode]]")
+	balloon_alert(user, "mode: [ops[mode]]")
 
 /obj/item/door_remote/interact_with_atom(atom/interacting_with, mob/living/user, list/modifiers)
 	if(!istype(interacting_with, /obj/machinery/door) && !isturf(interacting_with))
 		return NONE
 	return ranged_interact_with_atom(interacting_with, user, modifiers)
 
+/obj/item/door_remote/omni
+	name = "omni door remote"
+	desc = "This control wand can access any door on the station."
+	department = "omni"
+	region_access = REGION_ALL_STATION
+	our_domain = list(	/area/station	)
+
+/obj/item/door_remote/captain
+	name = "command door remote"
+	desc = "A remote for controlling a set of airlocks. Despite its gaudy insignia denoting the Captain as its owner, some fine print\
+		indicates that its access is exclusively relegated to the Bridge and high-security command areas -- an additional byline\
+		specifically excludes Security from the high-security areas. Ironic."
+	department = "command"
+	region_access = REGION_COMMAND
+	our_domain = list(
+		/area/station,	// still restricted by limited accesses in REGION_COMMAND
+	)
+
+/obj/item/door_remote/chief_engineer
+	name = "engineering door remote"
+	desc = "A remote for controlling a set of airlocks. This one smells like burnt flesh and ozone."
+	department = "engi"
+	region_access = REGION_ENGINEERING
+	owner_trim = /datum/id_trim/job/chief_engineer
+	// doesn't need a domain because their specific high-security areas aren't on anyone else's trim but cap
+
+/obj/item/door_remote/research_director
+	name = "research director's door remote"
+	desc = "A remote for controlling a set of airlocks. This one is slightly misshapen, as if squeezed by a person possessing ludicrous strength."
+	department = "sci"
+	region_access = REGION_RESEARCH
+	owner_trim = /datum/id_trim/job/research_director
+	our_domain = list(
+		/area/station/ai_monitored/turret_protected/ai,
+		/area/station/ai_monitored/turret_protected/ai_upload_foyer,
+		/area/station/ai_monitored/turret_protected/ai_upload,
+	)
+
+/obj/item/door_remote/head_of_security
+	name = "warden's(?) door remote"
+	desc = "A remote for controlling a set of airlocks. This one smells like sweat, blood, resentment, and coffee.\
+		Someone appears to have tampered with the identifier."
+	department = "security"
+	region_access = REGION_SECURITY
+	owner_trim = /datum/id_trim/job/head_of_security
```

Comment:
HoS gets quite a bit of extra access to every department, will this wand give Warden the ability to open them all? Would be a massive buff to him especially since they work through camera consoles, he'd be essentially replacing the AI as security's personal door opener.